package.json

```diff
@@ -80,7 +80,7 @@
     "start:server": "nodemon --watch server.js --exec 'node server.js'",
     "commit:hash": "echo REACT_APP_UI_COMMIT_HASH=$(git rev-list -1 --all) > .env",
     "build": "yarn commit:hash && yarn build:css && yarn build:js && yarn postbuild",
-    "test": "yarn lint && react-scripts test --env=jsdom --coverage",
+    "test": "yarn lint && react-scripts test --updateSnapshot --env=jsdom --coverage",
```

Comment:
Should the snapshots be automatically updated whenever you run the tests?
Doesn't that negate the testing completely if it always updates the snapshots to the latest, or do the tests fail?

Also, I don't think running tests should update src files

Reply:
OK. 
My understanding was these tests fail if the UI for a particular page doesn't match anymore.
This failure lets the developer know something has changed and they have to either:

* change their code as they didn't mean to have the UI be updated
* update the snapshot as they are OK with the highlighted differences

Reply:
Ok, that's also a valid viewpoint. I'll change it to a manual step.